homeassistant/components/isy994.py

```diff
@@ -132,6 +134,20 @@ def _categorize_programs() -> None:
                         PROGRAMS[component].append(program)
 
 
+def _categorize_weather() -> None:
+    """Categorize the ISY994 weather data."""
+    global WEATHER_NODES
```

Comment:
Oooh, that's new, and useful. For the time being, I'd like to keep it the way it is because there are a couple other globals used in this way. For the purposes of a hotfix, I'd rather not change more than I have to. Once this is in dev and master, I'd be happy to fix the ISY component to do that.